test/cljs/cider/nrepl/middleware/cljs_ns_test.clj

```diff
@@ -13,10 +13,18 @@
 
   (testing "ns-vars op"
     (let [{:keys [ns-vars]} (session/message {:op "ns-vars"
-                                              :ns "cljs.reader"})]
+                                              :ns "cljs.core"})]
       (is (sequential? ns-vars))
       (is (every? string? ns-vars))))
 
+  (testing "ns-vars-with-meta op"
+    (let [ns-vars-with-meta (:ns-vars-with-meta
+                             (session/message {:op "ns-vars-with-meta"
+                                               :ns "cljs.core"}))]
+      (is (every? (comp map? second) ns-vars-with-meta))
+      (is (= (:+ ns-vars-with-meta)
+             {:arglists "(quote ([] [x] [x y] [x y & more]))"}))))
```

Comment:
This test should've failed. I'm not sure why it doesn't. Probably something to do with the cljs env, not really sure. It works when I have a actual cljs jar loaded in the repl though, it returns the expected `"macro"` param.